Makefile

```diff
@@ -6,15 +8,32 @@ build:
 up:
 	docker-compose up
 
+down:
+	$(ddown) --remove-orphans
+
 migrate:
-	docker-compose run --rm web python manage.py migrate
+	docker-compose run --rm web python manage.py migrate;
+	$(ddown)
```

Comment:
Nice. I would have left it on the same line probs but that's just a style preference